logos/src/logos/main.py

```
@@ -1741,11 +1768,12 @@ async def refresh_pipeline_runtime_state(
     This keeps queue state and active request tracking intact while making newly
     added providers/deployments/models available immediately.
     """
-    global _pipeline, _logosnode_facade, _azure_facade
-    if not _pipeline or not _logosnode_facade or not _azure_facade:
+    global _pipeline, _logosnode_facade, _azure_facade, _peer_facade
```

Flake8 (7.3.0)

[error] `global _pipeline` is unused: name is never assigned in scope

(F824)